management/server/activity/event.go

```diff
@@ -31,12 +34,13 @@ func (e *Event) Copy() *Event {
 	}
 
 	return &Event{
-		Timestamp:   e.Timestamp,
-		Activity:    e.Activity,
-		ID:          e.ID,
-		InitiatorID: e.InitiatorID,
-		TargetID:    e.TargetID,
-		AccountID:   e.AccountID,
-		Meta:        meta,
+		Timestamp:      e.Timestamp,
+		Activity:       e.Activity,
+		ID:             e.ID,
+		InitiatorID:    e.InitiatorID,
+		InitiatorEmail: e.InitiatorEmail,
```

Comment:
In the db we do not store for every events email address. In this case a pointer what could be nil express better the db structure. But definitely you are right it would be simpler a simple string. I will check what can I do.